### PR TITLE
The Bad Guys: tweaks

### PR DESCRIPTION
**Pull request by breville**
Some tweaks for The Bad Guys content:

- Support for a black action button on the homepage.
- Bold movie title on the homepage.
- Bold & italic movie title on the landing page heading.
- Bold movie title on the landing page body text.

### https://code.org/

Screen Shot 2022-04-07 at 9 41 43 AM

### https://code.org/thebadguys

Screen Shot 2022-04-07 at 9 31 56 AM

### CODE REVIEW

**Review by kelbyhawn — APPROVED** [review]

**Comment by kelbyhawn** [comment]
[The Bad Guys](https://trello.com/c/LFUR4e7L/1-the-bad-guys)

File: pegasus/sites.v3/code.org/views/homepage_hero.haml [hunk 1]
```diff
@@ -41,7 +42,7 @@
           = view :display_video_thumbnail, thumbnail_only: true, id: "mainvideo", video_code: entry[:youtube_id], image_url: entry[:image_url], play_button: 'center', letterbox: "false"
       - elsif entry[:type] == "text"
         .text{dir: "auto"}
-          = I18n.t(entry[:text])
+          != I18n.t(entry[:text], markdown: :inline)
```
Comment (Hamms) [inline pegasus/sites.v3/code.org/views/homepage_hero.haml]: I think this should either be

        .text{dir: "auto"}
          != I18n.t(entry[:text], markdown: true)

or

        %p.text{dir: "auto"}
          != I18n.t(entry[:text], markdown: :inline)

Comment (breville) [reply pegasus/sites.v3/code.org/views/homepage_hero.haml]: Oh interesting.  We tried the former and saw that it generated wrapper `<p>` tags but they weren't needed for our layout.  Are they important from a security point of view?

Comment (Hamms) [reply pegasus/sites.v3/code.org/views/homepage_hero.haml]: from an HTML validity point of view; `div`s should contain only flow content like paragraphs, not phrasing content like text

Comment (breville) [reply pegasus/sites.v3/code.org/views/homepage_hero.haml]: Ah fair enough.  I'll go back to the former then!

**Review by Hamms — APPROVED** [review]
LGTM!

File: pegasus/sites.v3/code.org/views/homepage_hero.haml (hunk 1 above)
Comment (Hamms) [reply pegasus/sites.v3/code.org/views/homepage_hero.haml]: oh, also: using the `:inline` option will only render the subset of markdown syntax that can validly be rendered inline, so no lists, headers, etc. Presumably not a problem for this content, but still a potential source of confusion

**Pull request merged**